spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
```

Comment:
- [ ] Please make sure your tests are passing :+1:  now they are failing 

![image](https://user-images.githubusercontent.com/75126481/160906669-bd795720-a40f-4e55-a569-2c6b773cd396.png)

Reply:
### Important message to the next reviewer :point_down: 

- I had some issues running the tests locally due to me ruby version so I decided to have a call with the student to make sure everything was actually running well but we did that after I submitted my review because the student was a bit not available  so please approve the project 

![image](https://user-images.githubusercontent.com/75126481/160911513-baba8c1d-aa18-46ad-a1fb-7dac6d3b0d8e.png)

Reply:
Thank you, @hiromataba !